task_schedule.cfg

```diff
@@ -43,7 +43,7 @@ alert       aca@head.cfa.harvard.edu
       exec 1: parse_cmd_load_gen.pl --dbi 'sybase' --server 'sybase' --verbose
       exec 1: update_load_seg_db.py --dbi 'sybase' --server 'sybase' --verbose
       exec 6: $ENV{SKA_SHARE}/cmd_states/update_cmd_states.py --dbi 'sybase' --server 'sybase' --h5file $ENV{SKA_DATA}/cmd_states/cmd_states.h5
-      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --verbose
+      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --touch_file $ENV{SKA_DATA}/timelines/sqlite_parsed_files.touch  --verbose 
```

Comment:
Sure.  I can also add a softlink from the current file to "sum_files_sybase.touch" or some such.  I was trying to improve upon the naming this time around, but didn't really do a great job for my audience.